### PR TITLE
Changes the accepted file extension for XML-Exports to "zip"

### PR DESCRIPTION
**Pull request by oolscireum**
Fixes: SE-12786

### CODE REVIEW

**Review by fhaScireum — APPROVED** [review]

**Review by mkeckmkeck — APPROVED** [review]

File: src/main/java/sirius/biz/jobs/batch/file/XMLExportJobFactory.java [hunk 1]
```diff
@@ -34,7 +34,7 @@ protected void collectParameters(Consumer<Parameter<?>> parameterCollector) {
 
     @Override
     protected void collectAcceptedFileExtensions(Consumer<String> fileExtensionConsumer) {
-        fileExtensionConsumer.accept("xml");
+        fileExtensionConsumer.accept("zip");
```
Comment (andyHa) [inline src/main/java/sirius/biz/jobs/batch/file/XMLExportJobFactory.java:37]: do we always enforce compression or should we accept both?

**Review by sabieber — APPROVED** [review]

**Review by mko-sci — APPROVED** [review]

File: src/main/java/sirius/biz/jobs/batch/file/XMLExportJobFactory.java (hunk 1 above)
Comment (oolscireum) [reply src/main/java/sirius/biz/jobs/batch/file/XMLExportJobFactory.java:37]: currently we do always enforce compression.
even when there is ".xml" in the targetpath.

Comment (sabieber) [reply src/main/java/sirius/biz/jobs/batch/file/XMLExportJobFactory.java:37]: Currently we seem to do so, would be cool to support both

Comment (idlira) [reply src/main/java/sirius/biz/jobs/batch/file/XMLExportJobFactory.java:37]: I like forcing it, and so avoiding GB-big files "poluting" our storage.

**Review by idlira — APPROVED** [review]

**Review by andyHa — APPROVED** [review]

**Pull request merged**